src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1082,7 +1112,8 @@ private bool ShouldExecuteOnAll<TResult>(ServerNode chosenNode, RavenCommand<TRe
                    selector.Topology?.Nodes?.Count > 1 &&
                    command.IsReadRequest &&
                    command.ResponseType == RavenCommandResponseType.Object &&
-                   chosenNode != null;
+                   chosenNode != null && 
+                   typeof(IRaftCommand) != command.GetType();
```

Comment:
This doesn't look correct? `command is IRaftCommand == false` ?

Reply:
done